distribution/openrct2.d.ts

```diff
@@ -2781,6 +2781,16 @@ declare global {
          */
         lostCountdown: number;
 
+        /**
+         * The guest's current ride or stall destination.
+         */
+        rideHeadedTo: number | null;
+
+        /**
+         * Sends a guest to a given ride, regardless of their interest in it.
+         */
+        sendToRide(rideId: number): void;
```

Comment:
Any reason to have this method instead of letting plugins set the `rideHeadedTo` variable? I feel like just having the one variable and not the method would simplify matters.

Reply:
I like this recommendation and have implemented it. There wasn't a uniform return in my first implementation - the getter returned a Ride*, but the setter took an int. I've changed it so both get/set an int.